service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,165 @@
+                    <then>
+                        <set field="orderHeaderDetails" from="[orderId:orderId,orderName:order.orderName,placedDate:order.placedDate,statusId:order.statusId,orderRevision:
+                     order.orderRevision,currencyUomId:order.currencyUomId,productStoreId:order.productStoreId,salesChannelEnumId:order.salesChannelEnumId
+                     ,grandTotal:order.grandTotal]"/>
+                    </then>
+                </if>
+            </iterate>
+        </actions>
+    </service>
+
+    <!-- @  6.Listing all Order Item details for the order with orderId, 100102 and orderPartSeqId, 01.-->
+    <service verb="get" noun="OrderItems">
+        <out-parameters>
+            <parameter name="orderItemList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId" value="100102"/>
+                <econdition field-name="orderPartSeqId" value="01"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- @ 8.Find all Order Part records assigned to the facility with ZIRET_WH value-->
+
+    <service verb="get" noun="OrderPartData">
+        <in-parameters>
+            <parameter name="facilityId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderId"/>
+            <parameter name="orderPartSeqId"/>
+            <parameter name="facilityId"/>
+            <parameter name="partName"/>
+            <parameter name="customerPartyId"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.order.OrderPart" value-field="order"/>
+            <set field="orderId" from="order.orderId"/>
+            <set field="orderPartSeqId" from="order.orderPartSeqId"/>
+            <set field="statusId" from="order.statusId"/>
+            <set field="vendorPartyId" from="order.vendorPartyId"/>
+            <set field="customerPartyId" from="order.customerPartyId"/>
+            <set field="facilityId" from="order.facilityId"/>
+            <set field="carrierPartyId" from="order.carrierPartyId"/>
+            <set field="shipmentMethodEnumId" from="order.shipmentMethodEnumId"/>
+            <set field="partTotal" from="order.partTotal"/>
+            <set field="priority" from="order.priority"/>
+            <set field="disableShippingCalc" from="order.disableShippingCalc"/>
+            <set field="disableTaxCalc" from="order.disableTaxCalc"/>
+        </actions>
+    </service>
+
+    <!--  @ 9.Finding the count of order parts for the customer Party Id, “CustJqp”.-->
+
+    <service verb="get" noun="OrderPartsCount">
+        <out-parameters>
+            <parameter name="orderPartsCount" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartsCount">
```

Comment:
Here it returns the details of the order parts for the customer CustJqp.
We wish to get the count of order parts. Please explore how can we get the count.

Reply:
completed